src/widget/wstarrating.cpp

```diff
@@ -87,6 +92,32 @@ void WStarRating::mouseMoveEvent(QMouseEvent *event) {
     }
 }
 
+void WStarRating::slotStarsUp(double v) {
+    if (!m_pCurrentTrack) {
+        return;
+    }
+    if (v > 0 && m_starRating.starCount() < m_starRating.maxStarCount()) {
+        int star = m_starRating.starCount() + 1;
+        qDebug() << "   stars " << m_starRating.starCount() << " > " << star;
```

Comment:
why we have extra space around stars? qDebug() already places spaces around << added strings.

Reply:
This is left-over debug output I will remove

Reply:
white the leading whitespace the output is easier to spot in the log stream